.github/workflows/pull-request-swift-toolchain-cirun.yml

```diff
@@ -17,8 +17,8 @@ jobs:
     uses: ./.github/workflows/swift-toolchain.yml
     with:
       create_release: false
-      default_runner: "cirun-win11-23h2-pro-x64-16-2024-05-17"
-      compilers_runner: "cirun-win11-23h2-pro-x64-64-2024-05-17"
+      default_runner: "cirun-win11-23h2-pro-x64-16-2024-05-17--${{ github.run_id }}"
```

Comment:
Why the `--` separator here but not near the beginning (`cirun--`)?

Reply:
From: https://docs.cirun.io/reference/unique-runner-labels.    Cirun ignores everything after `--`.  Everything before `--` is matched against the labels `.cirun.yml`.